server/src/frameworks/Hardhat/HardhatProject.ts

```diff
@@ -253,6 +243,32 @@ export class HardhatProject extends Project {
     return resolveActionsFor(this.serverState, diagnostic, document, uri);
   }
 
+  private _sendToChild(message: Message) {
+    this._assertWorkerExists();
+    this._assertWorkerIsRunning();
+    this._assertWorkerConnected();
```

Comment:
I added an assertion to check that the child process is still connected.

Reply:
We always want to check whether the child process is connected before calling send.